main.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python3 -m gcn_classic_text_to_json.notices.calet.__init__
```

Comment:
How is this code intended to be run? Is it something that will be called from the command line?

Reply:
Yes, this is how the code is intended to be run for now. It needs to be called from the command line with the repo directory as your root. This was just something quick that I got up so I could start testing if the conversions were accurate. If there are better ideas for how to call the subpackages, I can change it.